packages/scripts/utils/config.js

```diff
@@ -184,6 +183,10 @@ function getWebpackEntryPoints() {
 		return JSON.parse( process.env.WP_ENTRY );
 	}
 
+	if ( ! hasProjectFile( 'src' ) ) {
```

Comment:
```suggestion
	// Continue only if the `src` directory exists
	if ( ! hasProjectFile( 'src' ) ) {
```

When reading `hasProjectFile()` you may assume it's looking for a file, so maybe add a comment for clarity?

Reply:
Yes, good point. The name is a bit misleading, but that's a simple wrapper over `existsSync` from 'fs` 👍🏻